# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/cookiebanners/nsICookieBannerTelemetryService.idl
# Commit: 517a86043166
# Full Hash: 517a86043166302c58e961b95fc0876d07de6dc6
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2024-03-29 04:51:52
# Regressor Bug: 1874741
# File Overlap Count: 1
# Description:
#   Bug 1874741 - Part 3: Implement the telemetry to collect GDPR choice on Google Search. r=anti-tracking-reviewers,pbz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205443
# ==============================================================================

diff -r ca7422bd712f -r 517a86043166 toolkit/components/cookiebanners/nsICookieBannerTelemetryService.idl
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/toolkit/components/cookiebanners/nsICookieBannerTelemetryService.idl	Thu Mar 28 20:46:34 2024 +0000
@@ -0,0 +1,13 @@
+/* -*- Mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "nsISupports.idl"
+
+/**
+ * Service singleton which collect cookie banner telemetry.
+ */
+[scriptable, builtinclass, uuid(56197e18-d144-45b5-9f77-84102f064462)]
+interface nsICookieBannerTelemetryService : nsISupports {};
